src/org/zaproxy/zap/extension/websocket/manualsend/WebSocketReopenSend.java

```diff
@@ -0,0 +1,320 @@
+package org.zaproxy.zap.extension.websocket.manualsend;
```

Comment:
Should have the standard license header.
https://github.com/zaproxy/zap-admin/blob/master/docs/headers/license.java (don't forget to replace `$YEAR`)